src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexInterpreter.cs

```diff
@@ -329,15 +329,20 @@ private bool MatchRef(int index, int length, ReadOnlySpan<char> inputSpan)
 
         private void Backwardnext() => runtextpos += _rightToLeft ? 1 : -1;
 
-        protected override bool FindFirstChar() =>
-            _code.FindOptimizations.TryFindNextStartingPosition(runtext!, ref runtextpos, runtextbeg, runtextstart, runtextend);
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Nit, I believe we prefer to avoid AggressiveInlining unless we have data that it makes a real difference, or at least we have observed that it is not getting inlined when it obviously should be. I would assume these would get inlined without us adding hints here.

Reply:
I expect these won't be inlined regardless of the attribute, at least not without PGO.  These are virtuals, invoked only by a method on the base.

Reply:
Good to know, I'll remove the attribute then.